politeiawww/politeiawww.go

```diff
@@ -66,6 +68,11 @@ type wsContext struct {
 	done          chan struct{} // SHUT...DOWN...EVERYTHING...
 }
 
+type addressWatcherContext struct {
```

Comment:
Could use a comment.  It's not clear what this is used for.

Reply:
Unused now, removed.